src/sentry/api/validators/sentry_apps/schema.py

```diff
@@ -130,14 +130,24 @@
             },
             "required": ["type", "link", "create"],
         },
-        "alert-rule-action": {
+        "alert-rule-settings": {
             "type": "object",
             "properties": {
-                "type": {"type": "string", "enum": ["alert-rule-action"]},
+                "type": {"type": "string", "enum": ["alert-rule-settings"]},
+                "uri": {"$ref": "#/definitions/uri"},
                 "required_fields": {"$ref": "#/definitions/fieldset"},
                 "optional_fields": {"$ref": "#/definitions/fieldset"},
             },
-            "required": ["type", "required_fields"],
+            "required": ["type", "uri", "required_fields"],
+        },
+        "alert-rule-action": {
```

Comment:
Does modifying this schema file have no effect on existing 'alert-rule-actions' or whatever? 

Are there no places in our app (or other integrations) that will keyError when they try to access `alert-rule-action[properties][required_fields]` besides tests? It might just be my misunderstanding of what this file does but changes to existing fields on schema's make me nervous.

Reply:
@leeandher nah this schema is for validating the UI Component schema for a Sentry App. We currently do not have Alert Rule UI Components, so changing this won't break anything in the UI. If anyone did have this in their schema, they will need to update if they want to change anything else in their Sentry App settings. That's acceptable since, this was not made public and did not have a function until now.

Reply:
Cool, thanks 👍